environments/dev-env/install_edge-mc.sh

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+stage=""
+clusters="florin guilder"
+verbosity=0
+
+while (( $# > 0 )); do
+    if [ "$1" == "--stage" ]; then
+        stage=$2
+        shift
+    elif [ "$1" == "--clusters" ]; then
+        clusters=$2
+        shift
+    elif [ "$1" == "-v" ]; then
+        verbosity=1
+    fi 
+    shift
+done
+
+# Check if docker is running
+if ! docker ps > /dev/null
+then
+  echo "Docker Not running ...."
+  exit
+fi
+
+# Check go version
+go_version=`go version | { read _ _ v _; echo ${v#go}; }`
+
+function ver { printf "%03d%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
+if [ $(ver $go_version) -lt $(ver 1.19) ]; then
+    echo "Update your go version"
+fi
+
+# Deleting kind clusters
+for c in ${clusters[@]}
+do 
+  if [ $(kind get clusters | grep $c) > /dev/null 2>&1 ]; then
+     echo "kind cluster $c already exists - deleting it ...."
+     kind delete cluster --name $c > /dev/null 2>&1
+  fi
+done
+
+# KCP is an older kcp-edge deployment is already running
+process_running() {
+  SERVICE="$1"
+  if pgrep -f "$SERVICE" >/dev/null
+  then
+      echo "running"
+  else
+      echo "stopped" 
+  fi
+}
+
+# Check kcp-edge is already running
+if [ $(process_running kcp) == "running" ]
```

Comment:
Will this match too much --- all processes that happen to have "kcp" in their name?